docs/components/MessagingAttachmentView.jsx

```diff
@@ -0,0 +1,264 @@
+  // Add new props
+  _renderProps() {
+    return (
+      <>
+        <PropDocumentation
+          title="<MessagingAttachment /> Props"
+          availableProps={[
+            {
+              name: "attachmentID",
+              type: "string",
+              description: "Unique ID used to identify this attachment",
+              // optional: false,
+            },
+            {
+              name: "errorMsg",
+              type: "string",
+              description: "Optional error message, used when there is an error uploading, etc.",
+              optional: true,
+            },
+            {
+              name: "icon",
+              type: "React.ReactNode",
+              description:
+                "A ReactNode to use as the icon for the Attachment. Generally will be a call to <FileAttachmentIcon /> ",
+              // optional: false,
+            },
+            {
+              name: "onClickAttachment",
+              type: "(attachmentID: string) => void",
+              description: "Callback to be called upon clicking the Attachment",
+              // optional: false,
+            },
+            {
+              name: "onRemoveAttachment",
+              type: "(attachmentID: string) => void",
+              description: "Callback to be called upon clicking the Remove Attachment (X) button",
+              optional: true,
+            },
+            {
+              name: "title",
+              type: "string",
+              description: "Title of the file",
+              // optional: false,
+            },
+            {
+              name: "subtitle",
+              type: "string",
+              description: "Optional subtitle, generally used for upload status, filesize, etc",
+              optional: true,
+            },
+            {
+              name: "isUpload",
+              type: "boolean",
+              description:
+                "Optional boolean for whether this Attachment is a draft/upload or it is a sent Attachment",
+              defaultValue: "false",
+              optional: true,
+            },
+            {
+              name: "uploadComplete",
+              type: "boolean",
+              description: "Optional boolean for whether this Attachment has finished uploading",
+              defaultValue: "false",
+              optional: true,
+            },
+            {
+              name: "uploadError",
+              type: "boolean",
+              description:
+                "Optional boolean for whether there was an error while uploading the Attachment",
+              defaultValue: "false",
+              optional: true,
+            },
+          ]}
+          className={cssClass.PROPS}
+        />
+        <PropDocumentation
+          title="<FileAttachmentIcon /> Props"
+          availableProps={[
+            {
+              name: "fileType",
+              type: '"audio" | "catchall" | "doc" | "image" | "pdf" | "ppt" | "video" | "xls"',
```

Comment:
Which icon is the `"catchall"` for? What exactly does it cover? 😮

Reply:
![Screen Shot 2021-05-13 at 3 36 57 PM](https://user-images.githubusercontent.com/54862564/118195993-1843de00-b401-11eb-9245-cdbc42106632.png)

This one, its just like a fallback